src/lat/lattice-functions.h

```diff
@@ -306,6 +306,21 @@ void ComposeCompactLatticeDeterministic(
     fst::DeterministicOnDemandFst<fst::StdArc>* det_fst,
     CompactLattice* composed_clat);
 
+/// This function returns true if the CompactLattice clat has unique
+/// left phone context of length n, and false if it does not.
+bool HasUniquePhoneContext(const CompactLattice &clat,
+       const TransitionModel &tmodel, int32 n);
```

Comment:
Actually I just checked what composition does, not how it does it. I will spend some time studying it.
So now that we don't need phone expansion, should I leave the functions be, or delete them ?